### ISSUE TITLE
Output data in any suitable semi-structured format w/out templates

### ISSUE THREAD

**Issue by briandominick** [DocOps/liquidoc-gem#5]
LiquiDoc can ingest data from JSON, XML, YAML, and CSV, but in order to output in these formats the user currently has to actually create a template. Since Ruby can convert data to these formats (with way less human error and frustration), let's enable automatic conversion. Maybe something like:
```yaml
- action: parse
  data: _data/testdata.xml
  output: build/frontend/testdata.json
```
Some conversions won't necessarily work, such as anything nested to CSV, so we should test for depth on that one. Others should convert pretty cleanly.

**Comment by briandominick** [comment]
Due to XML and CSV output being more complex in Ruby than JSON or YAML output, I'm only implementing the latter two for now. I can see good use cases for both XML and CSV output, but it will have to wait until a more pressing need.

**Comment by briandominick** [comment]
Should have been closed long ago due to PR #28.
